preloader.webpack.config.js

```diff
@@ -68,15 +68,19 @@ module.exports = {
         },
       },
       {
-        type: 'javascript/auto',
+        type: "javascript/auto",
         test: /\.mjs$/,
-        use: []
-      }
+        use: [],
+      },
     ],
   },
   resolve: {
     alias: {
       "~": path.resolve(__dirname, "src"),
+      "@ui": path.resolve(__dirname, "ui-lib"),
```

Comment:
Is it really imported from `src/preloader/index.js`?

Reply:
forgot about that one its the colors for the loader animation right :thinking:

Reply:
done i added a @styles to import things from that folder directly :+1: